CONTRIBUTING.md

```diff
@@ -93,20 +128,17 @@ To follow [Go testing conventions](https://golang.org/pkg/cmd/go/internal/test/)
   
 #### Integration testing
 
-You can run integrations in different environment- local, demo, docker with either OSS or DAP deployments. 
-All you need to do is run `./bin/test_integration` with the proper flags.
-
-Run on docker: `--docker`
-
-Run demo: `--demo`
+Our integration tests can be run against either a GKE / Openshift remote cluster. To do so, run `./bin/test_integration` and add the proper flags. 
```

Comment:
this is not the case anymore, correct? we need to run `./bin/start`

Reply:
great catch